metrics/confusion_matrix/confusion_matrix.py

```diff
@@ -41,6 +42,13 @@
         >>> results = confusion_matrix_metric.compute(references=[0, 1, 2, 0, 1, 2], predictions=[0, 1, 1, 2, 1, 0])
         >>> print(results)  # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE
         {'confusion_matrix': array([[1, 0, 1], [0, 2, 0], [1, 1, 0]][...])}
+
+    Example 2-Multilabel scenario
+        >>> you must pass (config_name="multilabel") to the load method
+        >>> confusion_matrix_metric = evaluate.load("confusion_matrix")
```

Comment:
why are you not passing it to the load method? the doctest will fail that way.

Reply:
Done ✅